openquake/qa_tests_data/event_based_risk/case_4/job_hazard.ini

```diff
@@ -0,0 +1,44 @@
+[general]
+description = Event Based Hazard for Turkey reduced
+calculation_mode = event_based
+random_seed = 323
+
+[geometry]
+structural_vulnerability_file = models/vuln/structural_vulnerability_model.xml
+exposure_file = models/exp/exposure.xml
+
+[site_params]
+# site_model_file = models/site/site_model.xml
```

Comment:
I am going to use that for the next ticket: https://bugs.launchpad.net/oq-engine/+bug/1471100